packages/db/queries/get_snapshots_with_cursor.sql

```diff
@@ -0,0 +1,30 @@
+-- name: GetSnapshotsWithCursor :many
+-- We must convert NULL alias into empty string, because sqlc will not generate string pointer to it and it will crash when NULL is received
+SELECT COALESCE(ea.alias, '') as alias, sqlc.embed(s), sqlc.embed(eb)
+FROM "public"."snapshots" s
+JOIN "public"."envs" e ON e.id = s.env_id
+LEFT JOIN LATERAL (
+    SELECT alias FROM "public"."env_aliases" ea
```

Comment:
maybe aggregate them to array and return the first in API, it would be better prepared if we decide to support multiple aliases publicly, also it maybe solves the problem with `null`?

Reply:
Added. I aggregate them directly inside lateral because otherwise i would need to group by all fields manually